data/geany.glade

```diff
@@ -7228,7 +7228,7 @@
                                   <object class="GtkMenuItem" id="goto_tag_declaration1">
                                     <property name="visible">True</property>
                                     <property name="can_focus">False</property>
-                                    <property name="label" translatable="yes">Go to T_ag Declaration</property>
+                                    <property name="label" translatable="yes">Go to Symbol Dec_laration</property>
```

Comment:
This is unintentional - I was playing with moving the goto definition/declaration menu entries out of the submenu directly into the Search menu to make it more discoverable (because this is one of the greatest Geany features for me) and this is a leftover of that. Will fix.

After I started looking at the menus I started thinking they might need some more reorganisation - I might try to do something about it in the future.